pkg/webhook/validate_test.go

```diff
@@ -339,6 +339,28 @@ func TestStaticValidateSriovNetworkNodePolicyWithInvalidVendor(t *testing.T) {
 	g.Expect(ok).To(Equal(false))
 }
 
+func TestStaticValidateSriovNetworkNodePolicyWithInvalidVendorDevMode(t *testing.T) {
+	os.Setenv("DEV_MODE", "TRUE")
```

Comment:
Please, be aware the `os.Setenv(...)` creates an order dependency in the test suite. If we can leverage go1.17 feature, you can use `t.Setenv(...)` to make sure the environment variable is cleaned after the test.

Otherwise, [here there are some solutions] for the env variable issue. The .3 should work well

Reply:
Changed to `t.SetEnv`